bolt/lib/Target/AArch64/AArch64MCPlusBuilder.cpp

```diff
@@ -148,6 +149,68 @@ class AArch64MCPlusBuilder : public MCPlusBuilder {
     return false;
   }
 
+  MCPhysReg getAuthenticatedReg(const MCInst &Inst) const override {
```

Comment:
This seems to be missing `AUTI[AB]SPPC`, `AUTI[AB]SPPCR` and corresponding `RETA[AB]*` variants.

Reply:
Thanks for noticing! The original code here was written before these instructions were fully defined...
I've added them now in commit `f44f9bf`, but still need to write regression tests for these instructions.

Reply:
The regression tests have been added in the latest commit, I believe the work related to this comment is now complete.